apps/landingPage/components/Info/ResponsiveText.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import React from 'react';
+import { Text } from '@kiwicom/orbit-components/lib/';
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../../mediaQueriesConfig';
+
+const text =
+  "PLACEHOLDER!!!!!You've had a look at Tequila, but you've preffered yourTequila Tequila mixed with something else. Just grabMargarita.You've had a look at Tequila, but you've always prefferedyour Tequila mixed with something else.u've had a look at Tequ";
+
+export default function ResponsiveText() {
+  return (
+    <WrapperText>
+      <Mobile>
+        <Text size="large" align="left">
+          {text}
+        </Text>
+      </Mobile>
+      <Desktop>
+        <Text size="large" align="center">
+          {text}
+        </Text>
+      </Desktop>
+    </WrapperText>
+  );
```

Comment:
With this approach there will be the same text two times in HTML, shouldn't be better do it with nested styles? 
something like:
```
 @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) {
	  > p {
	      text-align: center;
	    }
```

Reply:
Much better like that!